src/js/components/form-elements/FieldArrayComponent.jsx

```diff
@@ -123,9 +123,9 @@ class FieldArrayComponent extends Component {
                         <div>
                           {this.props.translate(config.headerTooltip, config.headerDefaultTooltip)}
                         </div>
-                      ) : (config.label &&
+                      ) : (headerLabel &&
```

Comment:
I'm a bit scared of such changes to generic components. Have you made sure nothing else broke?

Reply:
+1, @alannadolny?

Reply:
yup, it works